domrender/renderer-js-instructions.go

```diff
@@ -123,12 +123,30 @@ func (il *instructionList) checkLenAndFlush(l int) error {
 				return err
 			}
 			err = il.checkLen(l)
+			if err == errDoesNotFit {
+				size := computeRequiredSize(il.pos + l)
```

Comment:
```suggestion
				size := computeRequiredSize(l)
```
After a succesful call to il.flush(), il.pos will be 0.